code/processes/wdb.q

```diff
@@ -375,7 +375,7 @@ informgateway:{[message]
 	$[count gateways:.servers.getservers[`proctype;gatewaytypes;()!();1b;0b];
 	   [
 		   {.[@;(y;x);{.lg.e[`informgateway;"unable to run command on gateway"];'x}]}[message;] each exec w from gateways;
-		   .lg.o[`informgateway;"the message - ", message, " was sent to the gateways"]
+		   .lg.o[`informgateway;raze "the message - ",(1# string message), " was sent to the gateways"]
```

Comment:
Just use -3! here 

,(-3!message)," was ...

Reply:
Is `.Q.s` preferred over `-3!`? https://code.kx.com/q/ref/internal/#-3x-qs

Reply:
Yes, it should be consistent even if the internal function changes.